SkylineToolsStore/src/org/labkey/skylinetoolsstore/view/SkylineToolDetails.jsp

```diff
@@ -410,8 +406,7 @@ a.banner-button-small{
 <% } %>
         </div>
 
-        <a class="banner-button-small" style="float:left; margin-top:2px; margin-left:2px;" href="/labkey/project/home/software/Skyline/tools/Support/<%=h(tool.getName())%>/begin.view?" target="_blank">Support Board</a>
-
+        <button class="banner-button-small" href="/labkey/project/home/software/Skyline/tools/Support/<%=h(tool.getName())%>/begin.view?" target="_blank">Support Board</button>
```

Comment:
Would this work for href?
href<%=h(urlProvider(ProjectUrls.class).getBeginURL(getContainer().getChild("Support").getChild(tool.getName())))%>
Perhaps SkylineTool could have a getSupportContainer method. Also, we no longer need '?' at the end of URLs that don't have query parameters.